app/controllers/passwords_controller.rb

```diff
@@ -1,17 +1,18 @@
-class PasswordsController < Clearance::PasswordsController
+class PasswordsController < ApplicationController
   include MfaExpiryMethods
   include WebauthnVerifiable
   include SessionVerifiable
 
   before_action :validate_confirmation_token, only: %i[edit otp_edit webauthn_edit]
   after_action :delete_mfa_expiry_session, only: %i[otp_edit webauthn_edit]
 
-  # By default, clearance expects the token to be submitted with the password update.
-  # We already invalidated the token when the user became verified by token(+mfa).
-  skip_before_action :ensure_existing_user, only: %i[update]
-  # Instead of the token, we now require the user to have been verified recently.
+  # Instead of the token for update, we require the user to have been verified recently.
   verify_session_before only: %i[update]
 
+  def new
+    render template: "passwords/new"
```

Comment:
I don't think an explicit `render` is needed here

Reply:
Good point, copied this from clearance.